packages/nextjs/test/clientSdk.test.ts

```diff
@@ -150,13 +150,28 @@ describe('Client init()', () => {
         expect(browserTracingIntegration?.name).toBe('BrowserTracing');
       });
 
-      it('does not add `browserTracingIntegration()` integration if tracing not enabled in SDK', () => {
+      it('also adds `browserTracingIntegration()` integration if tracing not enabled in SDK (only build time)', () => {
```

Comment:
m: Instead of modifying this test I would simplify this one and the one above to simply assert that the integration was added. No point to test an invariant imo.